hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/dao/GatewayServiceLogEventDao.java

```diff
@@ -36,11 +36,12 @@
     this.grpcContextBuilder = grpcContextBuilder;
     this.requestBuilder = requestBuilder;
     this.logEventConverter = logEventConverter;
+    this.serviceConfig = serviceConfig;
 
     this.gatewayServiceStub =
         GatewayServiceGrpc.newFutureStub(
-                channelRegistry.forAddress(
-                    serviceConfig.getGatewayServiceHost(), serviceConfig.getGatewayServicePort()))
+            channelRegistry.forAddress(
```

Comment:
this formatting change will cause the build to break. We use the google style which is available as various ide plugins, but it's also applied and enforced via gradle (`gradlew spotlessApply` and `gradlew spotlessCheck`)

Reply:
I have just applied spotlessApply.